source/blog/2016-10-04-ovirt-provider-ovn.md

```diff
@@ -0,0 +1,163 @@
+
+* `openvswitch`
+* `openvswitch-ovn-common`
+* `openvswitch-ovn-host`
+* `python-openvswitch`
+
+These are available from the [OVS website](http://openvswitch.org/download/) or built using the code downloaded from the OVS repo (described below).
+
+The OVN-controller service can be started using the following command:
+
+    systemctl start ovn-controller
+
+After installing the driver and OVN, the OVN-controller must be configured. This can be done either using the vdsm-tool:
+
+    vdsm-tool ovn-config <OVN central IP> <local IP used for OVN tunneling>
+
+or by using the OVN command-line interface directly. For more information about OVN-controller setup, please check the
+[OVS documentation](http://openvswitch.org/support/dist-docs/).
+
+The command above should create OVN tunnels to other OVN controllers (if at least one other ovn-controller is present).
+Please verify that the tunnel has been created by issuing the following commands:
+
+`ip link` - the result should include a link called `genev_sys_ ...`
+
+`ovs-vsctl show` - the bridge `br-int` should include a port looking somewhat like this:
+
+    Port "ovn-6b2d2f-0"
+       Interface "ovn-6b2d2f-0"
+            type: geneve
+            options: {csum="true", key=flow, remote_ip="10.35.128.14"}
+
+
+Please check the OVN logs in case of problems. The relevant OVN logs are located in `/var/log/openvswitch/`.
+
+
+## oVirt OVN Provider
+
+The oVirt OVN provider is a proxy that the oVirt Engine uses to interact with OVN. It is delivered as an rpm that is to be installed on the host where OVN central is installed.
+
+The [oVirt OVN provider RPM](http://resources.ovirt.org/repos/mmirecki/ovirt-provider-ovn-0-1.noarch.rpm) is also available now. The latest version can always be downloaded and built from the repository as well. Once the rpm is downloaded, it can be installed with this command:
+
+    yum install ovirt-provider-ovn-0-1.noarch.rpm
+
+OVN requires OVN to be installed on the host (version 2.6 or later). The following OVN packages must be installed:
+
+* `openvswitch`
+* `openvswitch-ovn-common`
+* `openvswitch-ovn-central`
+* `python-openvswitch`
+
+These are also available from the [OVS website](http://openvswitch.org/download/) or built using the code downloaded from the OVS repo (described below).
+
+After installing the oVirt OVN provider, the admin needs to open up port 9696 in the firewall.
+This can be done manually or by adding the ovirt-provider-ovn firewalld service to the appropriate firewalld zone.
+
+    firewall-cmd --zone=<zone to add service to> --add-service=ovirt-provider-ovn --permanent
+    firewall-cmd --reload
+
+The zone currently selected as default can be obtained by executing:
+
+    firewall-cmd --get-default-zone
+
+After installation, the provider can be started as follows:
+
+    systemctl start ovirt-provider-ovn
```

Comment:
Enable the service in order to make it start at boot, use: systemctl enable ovirt-provider-ovn